modules/installation-initializing.adoc

```diff
@@ -130,6 +134,22 @@ endif::gcp[]
 ... Paste the pull secret that you obtained from the
 link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers] page.
 
+ifeval::["{context}" == "installing-gcp-user-infra"]
+.. Optional: If you do not want the cluster to provision compute machines, empty
```

Comment:
@wking, we don't have this option for AWS UPI. Do we want to add it there or remove it from GCP?